blocks/api/raw-handling/shortcode-converter.js

```diff
@@ -33,7 +33,7 @@ export default function( HTML ) {
 
 			const attributes = mapValues(
 				pickBy( transform.attributes, ( schema ) => schema.shortcode ),
-				( schema ) => schema.shortcode( match.shortcode.attrs ),
+				( schema ) => schema.shortcode( match.shortcode.attrs, match ),
```

Comment:
Passing all of `match` is intentionally broad for the sake of the discussion here. For this PR, only `match.content` is needed. Using `content` seems better than attempting to reconstruct the code from the shortcode parser's output (for which we'd still need more than just `match.shortcode.attrs`).